charts/gardener/operator/templates/deployment.yaml

```diff
@@ -76,15 +79,19 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
         - --config=/etc/gardener-operator/config/config.yaml
-        {{- if or .Values.env .Values.imageVectorOverwrite .Values.componentImageVectorOverwrites }}
+        {{- if or .Values.env .Values.imageVectorOverwrite .Values.componentImageVectorOverwrites .Values.chartsImageVectorOverwrite }}
         env:
         {{- if .Values.imageVectorOverwrite }}
         - name: IMAGEVECTOR_OVERWRITE
-          value: /charts_overwrite/images_overwrite.yaml
+          value: /imagevector_overwrite/images_overwrite.yaml
         {{- end }}
         {{- if .Values.componentImageVectorOverwrites }}
         - name: IMAGEVECTOR_OVERWRITE_COMPONENTS
-          value: /charts_overwrite_components/components.yaml
+          value: /imagevector_overwrite_components/components.yaml
+        {{- end }}
+        {{- if .Values.chartsImageVectorOverwrite }}
+        - name: IMAGEVECTOR_CHARTS_OVERWRITE
```

Comment:
Why not `IMAGEVECTOR_OVERWRITE_CHARTS` similar to components? The variable in the Values is named that too.